website/content/docs/bloop-basics.md

```diff
@@ -51,52 +52,134 @@ Bloop is an agnostic build server that makes hot compiler accessible to all
-the fastest of the workflows no matter what the internal architecture of your
-build tool is.
+Build tool is a concept used to refer to developer tools that offer
+their users a way to write the logic of their build. They usually have a task
+engine, a story for caching and incremental execution, a host DSL to
+configure the build tool, a plugin ecosystem and other related functionality.
 
-#### Runtime independent
+This generic model allow build tools to express any build logic a user may
+desire. This model usually comes with a trade off between speed and other
+properties like correctness, usability and extensibility.
 
-Test or run your Scala application on the JVM, Scala.js and Scala Native.
-Bloop interfaces with the API of every runtime that the Scala ecosystem
-supports so that external tools don't have to.
+Bloop fundamentally lacks such a model because it doesn't aim to solve the
+same problems a build tool does! The problems that bloop aims to address are:
 
-#### Command line friendly
+* Compiling, testing and running code in the fastest way possible.
+* Provide the backbone so that other tools can build upon it.
 
-The primary way to interact with bloop is directly from the command line
-instead of through a shell or repl. This workflow is more familiar to how
-most of developer tools work (for example, Maven, Gradle or sbt) and allows
-you to reuse the same terminal for other tasks.
+It can then be used by Scala developers to be more productive and by tooling
+authors to write their tools faster and more reliably. As a result, many
+build tools, IDEs and client can reuse Bloop to have a fast developer
+workflow (and avoid manually integrating with Scala and Java incremental
+compilers).
+
+#### A note on the future of build tools
+
+It's an exciting time for Scala tooling!
 
-#### Simplicity
+There is a lot of activity in the build tools field; many smart
+people are working on how to make build tools in the Scala ecosystem faster,
+easier to use and more correct.
 
-You can think of bloop as a simple, end-user application that runs Bloop's
-core commands. The does not require you to learn a complex DSL or
-configuration format to interact with it. `--help` takes you a long way.
+Among these efforts we find organizations like Stripe or Wix working on
+[better Scala support for Bazel][scala-bazel] or Twitter getting excellent
+Scala support in [Pants][pants], and individuals like [Jan Christopher
+Vogt][cvogt] or [Li Haoyi][lihaoyi] working on [cbt] and [mill]
+respectively.
+
+All of these are exciting projects. In the future, we expect some of them to
+integrate with Bloop and benefit from a faster cross build-tool workflow! But
+we're also aware that they incidentally address the problem of compiling
+faster in different ways.
+
+With this in mind, we've created Bloop to primarly support all the existing
+build tools that companies and Scala open source developers use *today*. We
```

Comment:
Pedantry warning: "open-source" when it's an adjective!

Reply:
Awesome to know :smile: